cpp/example/test.c

```diff
@@ -130,9 +148,15 @@ int main(){
    free(b4.data);
    free_result(&res1);
 
-   void* subrange = subrange_new(st1, 0 , UINT64_MAX);
-   if (subrange == NULL){
-      perror("failed to init subrange");
+   ret = storage_insert(st1,time(NULL),&b2);
```

Comment:
Super nitpick comment xD
```suggestion
   ret = storage_insert(st1, time(NULL), &b2);
```